indra/tests/test_rlimsp.py

```diff
@@ -0,0 +1,30 @@
+from indra.sources import rlimsp
+
+
+def test_simple_usage():
+    rp = rlimsp.process_from_webservice('PMC3717945')
+    stmts = rp.statements
+    assert len(stmts) == 6, len(stmts)
+    for s in stmts:
+        assert len(s.evidence) == 1, "Wrong amount of evidence."
+        ev = s.evidence[0]
+        assert ev.annotations, "Missing annotations."
+        assert 'agents' in ev.annotations.keys()
+        assert 'trigger' in ev.annotations.keys()
+
+
+def test_ungrounded_usage():
+    rp = rlimsp.process_from_webservice('PMC3717945', with_grounding=False)
+    assert len(rp.statements) == 33, len(rp.statements)
+
+
+def test_grounded_endpoint_with_pmids():
+    pmid_list = ['16403219', '22258404', '16961925', '22096607']
+    stmts = []
+    for pmid in pmid_list:
+        rp = rlimsp.process_from_webservice(pmid, id_type='pmid',
+                                            with_grounding=False)
```

Comment:
Looks like this test is for the grounded endpoint but uses `with_grounding=False`

Reply:
Ah yes. The name did not change to keep up with reality.

Reply:
Then probably also worth adding at least one test of the grounded endpoint as well

Reply:
The first test is of the grounded endpoint. Do you mean a grounded endpoint with pmids?

Reply:
I added some more tests.

Reply:
Got it--didn't notice that with_grounding=True is the default in the first test